pkg/sql/schema_changer.go

```diff
@@ -1103,6 +1111,10 @@ func (s *SchemaChangeManager) Start(stopper *stop.Stopper) {
 						// changer doesn't get called again for a while.
 						sc.execAfter = timeutil.Now().Add(delay)
 					}
+
+					// TODO(vivek): shouldn't we only `break` if we actually ran a schema change
```

Comment:
@vivekmenezes could you comment on this?

Reply:
I don't think this is important.

Reply:
Can you be more specific? I.e. explain why we even pace schema changes in that way (why not do away with `execAfter`)? I'm happy to add commentary once you do.

Reply:
I believe the only reason to pace these is to allow different schema changes to execute on different nodes and distribute the load. But that was a thought a long time ago. You can safely move the break inside the if statement.